src/app/component/Menu/Menu.style.scss

```diff
@@ -0,0 +1,450 @@
+            }
+
+            @include desktop {
+                display: block;
+                @include subcategory-desktop-visible;
+            }
+        }
+
+        .Menu-Link {
+            @include desktop {
+                margin-left: 0;
+            }
+        }
+    }
+
+    &-SubItemWrapper {
+        .Menu-Link {
+            @include desktop {
+                color: var(--menu-second-level-color);
+                font-size: 14px;
+                text-transform: uppercase;
+            }
+
+            &:hover,
+            &:focus {
+                @include desktop {
+                    color: var(--menu-item-hover-color);
+                    text-decoration: none;
+                }
+            }
+        }
+    }
+
+    &-SubLevelDesktop {
+        .Menu-ItemList {
+            display: block;
+            padding: 0;
+
+            &_isBanner {
+                display: grid;
+                grid-column-gap: 10px;
+                grid-template-columns: repeat(2, 1fr);
+
+                @media (min-width: 1370px) {
+                    grid-template-columns: repeat(3, 1fr);
+                }
+            }
+        }
+
+        .Menu-Link {
+            color: var(--menu-third-level-color);
+            font-size: 11px;
+            margin-top: 20px;
+            text-transform: none;
+        }
+    }
+
+    &-Promotion {
+        @include desktop {
+            display: flex;
+            padding: 0 55px 40px;
+        }
+
+        .Menu-Link {
+            @include desktop {
+                margin-left: 0;
+                margin-right: 55px;
+            }
+        }
+    }
+
+    &-PageLink {
+        margin: 1.5rem 0;
+        text-align: center;
+
+        @include desktop {
+            font-size: 13px;
+            margin: 0;
+        }
+
+        .Menu-Link {
+            padding-left: 0;
+
+            &:hover,
+            &:focus {
+                @include desktop {
+                    color: var(--menu-item-hover-color);
+                }
+            }
+        }
+    }
+
+    &-Social {
+        display: flex;
+        justify-content: center;
+        margin-top: 2.5rem;
+
+        @include desktop {
+            align-items: center;
+            margin-left: 75px;
+            margin-top: 0;
+        }
+
+        h3 {
+            border: 0;
+            clip: rect(0 0 0 0);
+            height: 1px;
+            margin: -1px;
+            overflow: hidden;
+            padding: 0;
+            position: absolute;
+            width: 1px;
+        }
+
+        ul {
+            display: grid;
+            grid-auto-flow: column;
+            justify-items: center;
+            width: 14rem;
+
+            @include desktop {
+                grid-column-gap: 12px;
+                width: auto;
+            }
+        }
+
+        li {
+            width: 1.9rem;
+
+            @include desktop {
+                margin-bottom: 0;
+                width: 16px;
+            }
+        }
+    }
```

Comment:
I think we should move this to CMS block styles. There, we allow styling like this, by selector.